### ISSUE TITLE
Site does not render correctly with Dark Reader in Firefox

### ISSUE THREAD

**Issue by rpkelly** [bottlerocket-os/project-website#113]
**Path to the page with the problem (_do not_ include domain name):** Any documentation page

**Describe the problem:** When using Dark Reader in Firefox, site background is partially darkened, however none of the text is adjusted.

**Screen Shot, if possible** 
![image](https://user-images.githubusercontent.com/902032/230494399-d0840f7a-ff2b-477a-a869-931765af9d7e.png)

**Describe how the site should look?** As below:
![image](https://user-images.githubusercontent.com/902032/230494765-3ee80b32-bef0-4b0f-8562-bc2728d6102f.png)

**Comment by jpmcb** [comment]
Can confirm - getting a similar experience using Dark Reader on firefox

**Comment by stockholmux** [comment]
The Docsy template itself seemingly works fine with dark reader, so it has to be in our colour overrides somewhere. I'll dig in

**Comment by stockholmux** [comment]
I think I've got to the bottom of this:

The same website code served from Hugo's internal web server look fine in Dark Reader on Chrome:

![Screenshot 2023-04-11 at 7 48 16 AM](https://user-images.githubusercontent.com/1152927/231184785-13971e79-677f-4c7c-a532-a08520b4d767.png)

Same code served from the preview link provided to @rpkelly and @jpmcb is super broken (as illustrated above):
![Screenshot 2023-04-11 at 7 48 13 AM](https://user-images.githubusercontent.com/1152927/231185166-75ce2b23-09c5-4423-8545-c875fd218682.png)

The difference between the two: the preview link is on a subdirectory (`/foobar/`) while the hugo server is at the root (`/`). The preview link also has an authentication step followed by a HTTP redirect. One javascript dependency from Docsy (the template) is hard coded to the root (`/js/tabpane-persist.js`). 

![screenshot-debug-redact](https://user-images.githubusercontent.com/1152927/231187043-b655b1de-9606-4a57-8203-6e2d44de4e53.png)

It looks like this is causing any further JS to just stop loading. Looks like Dark Reader [relies partially on CSS injection as well as JS injection](https://github.com/darkreader/darkreader#using-dark-reader-for-a-website). Because the javascript vomits on the missing dependency, Dark Reader only partially loads and becomes super broken.

**Next steps**
1. I need someone to confirm that this related to the preview site server.
2. We should probably file an issue due to Docsy hard-coding a dependency to root (FWIW, they are moving from Bootstrap 4 to Bootstrap 5 at the time of writing and their development branch is not reliable for fixes, not sure if they would even accept a PR).

**Comment by stockholmux** [comment]
Confirmed that this issue is related to the preview site server on two different machines with different browsers.

Closing this as it's not related to the code itself, just a temporary/preview server.

**Comment by stockholmux** [comment]
Also, looks like this was fixed in https://github.com/google/docsy/pull/1233
